.github/workflows/aws_tfhe_fast_tests.yml

```diff
@@ -133,9 +126,15 @@ jobs:
         run: |
           echo "any_changed=true" >> "$GITHUB_OUTPUT"
 
+  check-user-permission:
+    if: github.event_name == 'pull_request_target'
+    uses: ./.github/workflows/check_triggering_actor.yml
+    secrets:
+      TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
   setup-instance:
     name: Setup instance (fast-tests)
-    if: github.event_name != 'pull_request' ||
+    if: github.event_name != 'pull_request_target' ||
       needs.should-run.outputs.any_file_changed == 'true'
     needs: should-run
```

Comment:
needs both the previous ones ?

Reply:
It only needs `check-user-permission` since they are sequential operations.